test/relations.test.js

```diff
@@ -4980,6 +4980,72 @@ describe('relations', function() {
       });
     });
 
+    it('should pass options when removed by id', function(done) {
+      const verifyOptions = function(ctx, next) {
+        if (!ctx.options || !ctx.options.verify) {
+          return next(new Error('options or options.verify is missing'));
+        }
+        return next();
+      };
+      Person.observe('before save', verifyOptions);
```

Comment:
I'm not too familiar with the observer codebase, so sorry if I misunderstand. We make the changes in `before delete` hook above, but we are not using it in the tests at all. Can you please explain the flow of the test and why we don't use it to verify the options object passed into it?